### ISSUE TITLE
Importing beatmap made in osu!lazer to osu!stable Doesn't work

### ISSUE THREAD

**Issue by sortedcord** [ppy/osu#13829]
I made a beatmap in osu!lazer using its editor and then I clicked on export package under files menu. It exported it as a osu score and then I tried to import it in osu!stable by dragging and dropping. Osu!stable then returned that the beatmap could not be loaded
**Screenshots or videos showing encountered issue:** 
OSU Stable:
![image](https://user-images.githubusercontent.com/37407370/124909778-7d364180-e008-11eb-8857-e790ac1ba3d0.png)

Lazer:
![image](https://user-images.githubusercontent.com/37407370/124909913-9e972d80-e008-11eb-83e1-3e37ab73334b.png)

**osu!lazer version:2021.703.0**

**Comment by bdach** [comment]
The beatmap attached to this issue is missing a `[TimingPoints]` section whatsoever. Will most likely be the cause.

@sortedcord I assume you did add a timing control point but it didn't save somehow? I imagine mapping without would be an insane endeavour.

This might be related to #12648 somehow. Needs further investigation.

**Comment by sortedcord** [comment]
Yeah, I made that beatmap without a timing control (It was my first time making a beatmap, so I didn't what was the correct way to do it). But that doesn't explain why it worked in lazer and not in stable though... After adding a timing control the beatmap does work correctly in stable.

**Comment by peppy** [comment]
The fix for this will likely be disallowing export until a timing point is set. Or moreso, disallowing hitobject placement.

We have a lot of work to do on the UX and guiding a user in the right direction when using the editor (and other game areas) for the first time. Even osu!(stable) is in a better state as far as this is concerned.
